flink-runtime/src/test/java/org/apache/flink/runtime/jobmanager/JobManagerMetricTest.java

```diff
@@ -50,9 +50,10 @@
 	public void testJobManagerMetricAccess() throws Exception {
 		Deadline deadline = new FiniteDuration(2, TimeUnit.MINUTES).fromNow();
 		Configuration flinkConfiguration = new Configuration();
-		
+
+		flinkConfiguration.setString(ConfigConstants.METRICS_REPORTER_CLASS, "org.apache.flink.metrics.reporter.JMXReporter");
 		flinkConfiguration.setString(ConfigConstants.METRICS_SCOPE_NAMING_JM_JOB, "jobmanager.<job_name>");
-		flinkConfiguration.setString(ConfigConstants.METRICS_JMX_PORT, "9060-9075");
+		flinkConfiguration.setString(ConfigConstants.METRICS_REPORTER_ARGUMENTS, "-port 9060-9075");
```

Comment:
the test still worked since it defaulted back to the default port range; it looked for a property `port`, but only `-port` was defined. We set ports here to avoid port conflicts; there is no real hard requirement for it.